float32[] previous_grasp_position
---
geometry_msgs/Pose target_grasp_pose